tests/system/providers/google/cloud/automl/example_automl_video_intelligence_tracking.py

```diff
@@ -22,49 +22,49 @@
 
 import os
 from datetime import datetime
-from typing import cast
+
+from google.cloud.aiplatform import schema
+from google.protobuf.struct_pb2 import Value
 
 from airflow.models.dag import DAG
-from airflow.models.xcom_arg import XComArg
-from airflow.providers.google.cloud.hooks.automl import CloudAutoMLHook
-from airflow.providers.google.cloud.operators.automl import (
-    AutoMLCreateDatasetOperator,
-    AutoMLDeleteDatasetOperator,
-    AutoMLDeleteModelOperator,
-    AutoMLImportDataOperator,
-    AutoMLTrainModelOperator,
-)
 from airflow.providers.google.cloud.operators.gcs import (
     GCSCreateBucketOperator,
     GCSDeleteBucketOperator,
     GCSSynchronizeBucketsOperator,
 )
+from airflow.providers.google.cloud.operators.vertex_ai.auto_ml import (
+    CreateAutoMLVideoTrainingJobOperator,
+    DeleteAutoMLTrainingJobOperator,
+)
+from airflow.providers.google.cloud.operators.vertex_ai.dataset import (
+    CreateDatasetOperator,
+    DeleteDatasetOperator,
+    ImportDataOperator,
+)
 from airflow.utils.trigger_rule import TriggerRule
 
 ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID", "default")
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT", "default")
 DAG_ID = "example_automl_video_track"
-GCP_PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT", "default")
-GCP_AUTOML_LOCATION = "us-central1"
-DATA_SAMPLE_GCS_BUCKET_NAME = f"bucket_{DAG_ID}_{ENV_ID}".replace("_", "-")
-RESOURCE_DATA_BUCKET = "airflow-system-tests-resources"
+REGION = "us-central1"
+VIDEO_DISPLAY_NAME = f"auto-ml-video-tracking-{ENV_ID}"
+MODEL_DISPLAY_NAME = f"auto-ml-video-tracking-model-{ENV_ID}"
 
-MODEL_NAME = "video_track_test_model"
-MODEL = {
-    "display_name": MODEL_NAME,
-    "video_object_tracking_model_metadata": {},
-}
+RESOURCE_DATA_BUCKET = "airflow-system-tests-resources"
+VIDEO_GCS_BUCKET_NAME = f"bucket_video_tracking_{ENV_ID}".replace("_", "-")
 
-DATASET_NAME = f"ds_video_track_{ENV_ID}".replace("-", "_")
-DATASET = {
-    "display_name": DATASET_NAME,
-    "video_object_tracking_dataset_metadata": {},
+VIDEO_DATASET = {
+    "display_name": f"video-dataset-{ENV_ID}",
+    "metadata_schema_uri": schema.dataset.metadata.video,
+    "metadata": Value(string_value="video-dataset"),
 }
-
-AUTOML_DATASET_BUCKET = f"gs://{DATA_SAMPLE_GCS_BUCKET_NAME}/automl/video_tracking.csv"
-IMPORT_INPUT_CONFIG = {"gcs_source": {"input_uris": [AUTOML_DATASET_BUCKET]}}
-
-
-extract_object_id = CloudAutoMLHook.extract_object_id
+VIDEO_DATA_CONFIG = [
+    {
+        "import_schema_uri": schema.dataset.ioformat.video.object_tracking,
+        # "gcs_source": {"uris": ["gs://cloud-samples-data/ai-platform-unified/video/traffic/traffic_videos_labels.csv"]},
```

Comment:
May I know what is this comment for? Should we remove it?

Reply:
Sure, thank you so much :)